src/Camille.RiotGames/Camille.RiotGames.csproj

```diff
@@ -17,6 +17,18 @@
     <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net45'">
+      <HintPath>$(NuGetPackageRoot)\microsoft.netframework.referenceassemblies.net45\1.0.3\build\.NETFramework\v4.5\System.Web.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'net461'">
+      <HintPath>$(NuGetPackageRoot)\microsoft.netframework.referenceassemblies.net461\1.0.3\build\.NETFramework\v4.6.1\System.Web.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Web" Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+      <HintPath>$(NuGetPackageRoot)\microsoft.netcore.app.ref\3.1.0\ref\netcoreapp3.1\System.Web.dll</HintPath>
+    </Reference>
+  </ItemGroup>
```

Comment:
I did what I could to keep the targets the same, but ultimately had to remove `netstandard1.1` and add references for `net45`, `net461`, and `netcoreapp3.1`.

It seems [this could be done more cleanly](https://stackoverflow.com/a/5710941/1843510) than what I have done in `Camille.RiotGames.csproj`, however none of those options exist for me in Rider or VS. Maybe they do for you, or it makes more sense to you?

Reply:
Hah good, always need an excuse to knock off the old versions